raiden/channel/netting_channel.py

```diff
@@ -431,7 +440,14 @@ def register_transfer_from_to(
         # nonce is changed only when a transfer is un/registered, if the test
         # fails either we are out of sync, a message out of order, or it's a
         # forged transfer
-        if transfer.nonce < 1 or transfer.nonce != from_state.nonce:
+        is_invalid_nonce = (
+            transfer.nonce < 1 or
```

Comment:
I would suggest to make this
```
transfer.nonce < 1 * NONCE_SCALING
```
and use `NONCE_SCALING = (2 ** 32)` in https://github.com/raiden-network/raiden/blob/master/raiden/channel/participant_state.py#L38, too.

Reply:
Removed the nonce scaling since it's not required.

Reply:
"Removed" as in "left out" here? Or did we remove nonce scaling globally?

Reply:
https://github.com/raiden-network/raiden/pull/913/commits/fa36dc61d17afab57381fd4cbc345ff0f44b2872